cmd/livenessprobe/main.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The Kubernetes Authors.
+Copyright 2019 The Kubernetes Authors.
```

Comment:
We don't update copyright  on updates (and no, I don't know why we have such strange rules).

Reply:
ACK, I'll update now